pyincore/models/fragilitycurveset.py

```diff
@@ -105,60 +105,6 @@ def calculate_limit_state(self, hazard_values: dict = {},
 
         return output
 
-    def calculate_limit_state_w_conversion(self, hazard, period: float = 0.0, std_dev: float = 0.0,
```

Comment:
@longshuicy Can you check this change and validate that removing these functions corresponds to deprecation?